### ISSUE TITLE
Papyrus integration tests are failing under Maven

### ISSUE THREAD

**Issue by djnemeth** [ELTE-Soft/txtUML#391]
Steps to reproduce:

1. Fix the *[release.target](https://github.com/ELTE-Soft/txtUML/blob/c94570311efb3ca1a1c8ee728cc0b6ef2daf912c/dev/releng/hu.elte.txtuml.target/release.target)* definition according to 82f64aa.
2. Run the *[clean-verify-release.launch](https://github.com/ELTE-Soft/txtUML/blob/c94570311efb3ca1a1c8ee728cc0b6ef2daf912c/dev/releng/hu.elte.txtuml.parent/clean-verify-release.launch)* configuration.

Interestingly, it seems that the aforementioned tests pass if they are run manually from the workspace as plugin tests. This might be a more general Maven configuration issue related to plugin unit tests, i.e. which require a separate Eclipse to be launched. See the attached error log for details: *[hu.elte.txtuml.export.papyrus.IntegrationTests.txt](https://github.com/ELTE-Soft/txtUML/files/374270/hu.elte.txtuml.export.papyrus.IntegrationTests.txt)*.

The reason why I suggested running the *clean-verify-release* launch config is that only the *release*  profile contains integration tests. The issue can also be reproduced with the default profile by running the *clean-verify* launch config, but only after enabling integration tests in the parent project's pom.

**Comment by djnemeth** [comment]
I suggest to postpone this issue to version 0.5.1.
